src/Controller/API/Administrator/EducationLevelAPI.php

```diff
@@ -1,47 +1,89 @@
 <?php
 
+/**
+ * This file is part of ESO system.
+ *
+ * (c) SBSystem 2018
+ *
+ * For the full license information, please see LICENSE.md at https://github.com/SBSystem/ESO/LICENSE.md
+ */
+
 namespace App\Controller\API\Administrator;
 
+use App\AppBundle\Factory\Creators\EducationLevelCreator;
 use App\Controller\API\Interfaces\CRUD;
 use Symfony\Component\HttpFoundation\JsonResponse;
 use Symfony\Component\HttpFoundation\Request;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 
-class EducationLevelAPI implements CRUD
+class EducationLevelAPI extends Controller implements CRUD
 {
     /**
      * This function is used to create education level
      */
     public function create()
     {
-        $request = new Request();
+        $jsonResponse = new JsonResponse();
 
-        $className = $request->request->get('class_name');
-        $classLevel = $request->request->get('class_level');
-        $educatorId = $request->request->get('educatorId');
+        if(isset($_POST['className']) && isset($_POST['classLevel']) && isset($_POST['educatorId'])){
+            $className = $_GET['className'];
+            $classLevel = $_POST['classLevel'];
+            $educatorId = $_POST['educatorId'];
 
-        $jsonResponse = new JsonResponse();
-        $jsonResponse->setData(array(
-            'educatorId' => $educatorId,
-            'classLevel' => $classLevel,
-            'className' => $className
-        ));
+            $educationCreator = new EducationLevelCreator();
+            $educationCreator->createObject($className, $educatorId, $classLevel);
+            $level = $educationCreator->getObject();
+
+            $doctrine = $this->getDoctrine()->getManager();
+            $doctrine->persist($level);
+            $doctrine->flush();
+
+            $jsonResponse->setData(array(
+                'educatorId' => $educatorId,
+                'classLevel' => $classLevel,
+                'className' => $className,
+                'httpStatusCode' => 200,
```

Comment:
po co to, skoro parę linijek nizej ustawiasz status code?

Reply:
Chcę, żeby user mógł sobie przejrzeć :)

Reply:
pytanie było o klucz 'httpStatusCode', bo niżej ustawiasz statusCode do Response'a

Reply:
`systemStatusCode` zwraca status z naszej własnej puli statusów
`httpStatusCode` zwraca status http żeby user mógł sobie przejrzeć (np. debuggujesz aplikację i sobie alertem wyświetlisz) - wszystko jest w docsach ;)